### PR TITLE
[Genomic Browser] TestPlan update

### PR DESCRIPTION
**Pull request by llevitis**
formatting & grammar error updates + updates to reflect new changes to the Genomic Browser

### CODE REVIEW

File: modules/genomic_browser/test/TestPlan.md [hunk 1]
```diff
@@ -27,50 +27,57 @@ The following permissions should be available in the database
 | genomic_browser_view_site | View Genomic Browser data from own site |
 | genomic_browser_view_allsites | View Genomic Browser data across all sites |
 | genomic_data_manager | Manage the genomic files |
+
+
+
 <br>
 ***
-#####For a user without neither genomic_browser_view_allsites nor genomic_browser_view_site
+#### For a user without neither genomic_browser_view_allsites nor genomic_browser_view_site
```
Comment (xlecours) [inline modules/genomic_browser/test/TestPlan.md]: /r missing maybe. The rendering is weird still

**Comment by driusan** [comment]
This needs @xlecours's approval, not mine. I don't know how the genomic browser is supposed to work.

File: modules/genomic_browser/test/TestPlan.md [hunk 2]
```diff
@@ -27,50 +27,54 @@ The following permissions should be available in the database
...
 - In the File tab of the genomic browser, there should be a *Upload File* button  
 
+*** 
+
 <a name="navigation">
-###Page navigation and display
 </a>
-***
+
+## Pave Navigation and Display
```
Comment (xlecours) [inline modules/genomic_browser/test/TestPlan.md]: Pave -> Page

File: modules/genomic_browser/test/TestPlan.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-######[Download](#data_download)  
-######[Upload](#data_upload)  
-####Help
-######[Help content](#help_content)
-######[Tooltips](#tooltips)
+## Genomic Browser - Test Plan
+
+## Table of Contents  
+
+### [Permissions](#permissions)
```
Comment (xlecours) [inline modules/genomic_browser/test/TestPlan.md]: The only links that are working are those with diferent name than their respective label. I think we should affix them with an '_a' or '_anchor'.

Comment (ridz1208) [inline modules/genomic_browser/test/TestPlan.md]: @xlecours If you tell me the changes to make I will update the PR since @llevitis is not here

**Comment by ridz1208** [comment]
@xlecours give it a try

**Comment by xlecours** [comment]
I tested it. Success

**Pull request merged**